pom.xml

```diff
@@ -602,6 +610,20 @@
         </profile>
         <profile>
             <id>kafka-0.8-dev</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-compiler-plugin</artifactId>
+                        <version>2.0.2</version>
+                        <configuration>
+                            <excludes>
+                                <exclude>com/pinterest/secor/timestamp/Kafka10MessageTimestamp.java</exclude>
```

Comment:
If 'exclude' tag works, would you be able to exclude Kafka8MessageTimestamp.java in kafka-0.10-dev build profile?  This way you don't have to introduce a dynamic class, everything is static, each build profile bundles a different set of java classes.